src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,24 @@
+package core.lottery;
+
+import core.balls.Ball;
+import core.colors.ColorSupplier;
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier;
+    private int number;
+        
+    public Lottery() {
+        colorSupplier = new ColorSupplier();
+    }
+     
+    public Ball getRandomBall() {
+        generateNumber();
+        return new Ball(number + 1, colorSupplier.getRandomColor());
+    }
+        
+    private void generateNumber() {
+        Random random = new Random();
+        this.number = random.nextInt(100);
```

Comment:
100 is a magic number - let's make it a constant field

Reply:
all done